packages/netlify-cms-core/src/components/MediaLibrary/MediaLibraryButtons.js

```diff
@@ -89,8 +89,14 @@ export class CopyToClipBoardButton extends React.Component {
 
   handleCopy = () => {
     clearTimeout(this.timeout);
-    const { path, draft, name } = this.props;
-    copyToClipboard(isAbsolutePath(path) || !draft ? path : name);
+    const { draft, name } = this.props;
+    let path = this.props.path;
+    if (isAbsolutePath(path)) {
+      copyToClipboard(path);
+    } else {
+      path = path.charAt(0) !== '/' ? `/${path}` : path;
```

Comment:
Hi @DenysLins, I think I wasn't clear with my last comment.
1. If `media_folder: static/upload` -> The copied path should not start with a `/`
2. If `media_folder: /static/upload` -> The copied path should start with a `/`

It seems to me that this code prepends a `/` if it doesn't already exist. Please correct me if I missed anything.

Reply:
I got it. I will work in this issue...